tests/end2end/features/conftest.py

```diff
@@ -199,6 +199,7 @@ def run_command(quteproc, httpbin, command):
 
     command = command.replace('(port)', str(httpbin.port))
     command = command.replace('(testdata)', utils.abs_datapath())
+    command = command.replace('(tmpdir)', str(tmpdir))
```

Comment:
Ah, you *do* use `(tmpdir)` in the test - in that case, I'm not sure if the `Given a new tmpdir` does anything at all, as the tempdir should already be created here